main.py

```diff
@@ -16,6 +16,7 @@
 parser.add_argument('-v', '--verbose', action="store_true", help="Enable verbose output")
 parser.add_argument('--output', action="store", default=None, help="Output file")
 parser.add_argument('--as-index', action="store_true", default=False, required=False, help="Outputs sitemap as index and multiple sitemap files if crawl results in more than 50,000 links (uses filename in --output as name of index file)")
+parser.add_argument('--sort-alphabetically', action="store_true", default=False, required=False, help="Sorts the output URLs alphabetically")
```

Comment:
Perhaps this shouldn't be an arg, but rather it is just always alphabetical?  I'm not sure there's any use case to have non-deterministic ordering.  @c4software WDYT?

Reply:
Hi,

I like the idea of having an argument but with default value to true.

Reply:
Yeah, that sounds good!